easybuild/easyconfigs/r/RELION/RELION-3.0.4_no_which_cmd.patch

```diff
@@ -0,0 +1,306 @@
+Don't use "which" to find the path to the program, it's either in that PATH or not.
```

Comment:
@akesandgren Can you clarify why this patch is needed? Why is it so bad that `which` is used to obtain the full path to the commands? No objections, just wondering...

Reply:
First of all it is not needed, the binaries are already in PATH, secondly I removed it because it was messing up the generated submit file causing errors.